packages/server/tests/acceptance/rpc_batch2.spec.ts

```diff
@@ -60,7 +60,7 @@ describe('@api-batch-2 RPC Server Acceptance Tests', function () {
     const ONE_TINYBAR = Utils.add0xPrefix(Utils.toHex(ethers.parseUnits('1', 10)));
     const ONE_WEIBAR = Utils.add0xPrefix(Utils.toHex(ethers.parseUnits('1', 18)));
 
-    const BASIC_CONTRACT_PING_CALL_DATA = '0x5c36b186';
+    const BASIC_CONTRACT_ESTIMATE_GAS_CALL_DATA = '0x6080604052348015600f57600080fd5b50609e8061001e6000396000f3fe608060405260043610602a5760003560e01c80635c36b18614603557806383197ef014605557600080fd5b36603057005b600080fd5b348015604057600080fd5b50600160405190815260200160405180910390f35b348015606057600080fd5b50606633ff5b00fea2646970667358221220886a6d6d6c88bcfc0063129ca2391a3d98aee75ad7fe3e870ec6679215456a3964736f6c63430008090033';
```

Comment:
nit: add a comment as to the function that it's calling to make this a bit more readable

Reply:
Done.

Reply:
I meant `0x5c36b186` mapped to `ping()` see https://www.4byte.directory/signatures/?bytes4_signature=0x5c36b186
What does the new hex map to so dev readers understand what this data content is